util_scripts/test-api-locally.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
this utility script allows to test the locally running service via direct api calls